### ISSUE TITLE
pydash.objects.omit doesn't support nested paths

### ISSUE THREAD

**Issue by beck3905** [dgilland/pydash#102]
I am trying to remove several fields from a json object with a nested structure using the `omit` function.  Given the following json:

```
obj = {
    "id": "123",
    "metadata": {
        "value": "abc"
    }
}
```

If I call `py_.omit(obj, "metadata.value")` I will receive the same `obj` as output with no fields removed when I would expect the following:

```
{
    "id": "123",
    "metadata": {}
}
```

  However, if I call `py_.omit(obj, "metadata") then I get the following object:

```
{
    "id": "123"
}
```

Can the omit method use nested paths?

**Comment by dgilland** [comment]
Currently, `omit` and `pick` do not work with nested paths but that's definitely something that could be changed (it works in Lodash).

Care to submit a PR for it? Otherwise, I'll try to get to it when I can.

**Comment by beck3905** [comment]
@dgilland I tried my best at a PR for this.  I created PR #104 with these changes.  Let me know if I need to tweak it before it can be merged in.  Thanks

**Comment by dgilland** [comment]
Closed via b5e9854a2e50409724b60f3df63dd9efdab71e5c.
